cli/src/pcluster/resources/custom_resources/custom_resources_code/delete_image_stack.py

```diff
@@ -21,6 +21,32 @@
 boto3_config = Config(retries={"max_attempts": 60})
 
 
+def _tag_ami(message_json):
+    # tag EC2 AMI
+    for ami in message_json["outputResources"]["amis"]:
```

Comment:
why not put this line into try ... except? So we can handle the key error of message_json["outputResources"]["amis"]

Reply:
done